### PR TITLE
tkt30_added_4_menu_pages_fixed_home_styling

### LINKED ISSUE #30: Menu Dropdown
Create menu pages for dining and takeout for both locations. Link each page to the Menus dropdown list.

### PR DESCRIPTION
**Pull request by vricci518**
Closes #30

Adjusted the navigation bar so Menus dropdown goes to 4 different pages.
Updated nav bar on Home and Menu page
Corrected the styling on the Home page

### CODE REVIEW

**Review by margeschrec — APPROVED** [review]

**Comment by vricci518** [comment]
Fixed menu dropdown #30

**Comment by vricci518** [comment]
Fixed Menu for dropdown #30

**Comment by billythakidd04** [comment]
I'm not going to do the review since it's already merged. But please make sure you wait for my approval before merging. Just to make sure. I promise I am trying to get to these within 12 hours, not always possible.

**Pull request merged**
